etna/models/nn/deepar_new/loss.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from torch import nn
+from torch.functional import F
+from torch.nn import GaussianNLLLoss
+
+
+class GaussianLoss(GaussianNLLLoss):
+    def __init__(self):
+        super().__init__(reduction="sum")
+
+    def forward(self, inputs, mean, std):
+        return super().forward(input=inputs, target=mean, var=std**2)
+
+    @staticmethod
+    def sample(mean, std):
+        with torch.no_grad():
+            return torch.normal(mean, std)
+
+
+class NegativeBinomialLoss(nn.Module):
```

Comment:
Better to inherit from [`_Loss`](https://pytorch.org/docs/stable/_modules/torch/nn/modules/loss.html#L1Loss) as all the pytorch losses